scripts/remove_invalid_social_entries.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+In preperation for OSF-8210 which will tighten social field validation, invalid entries must be removed. Pre OSF-8210 invalid social entries can be added and viewed directly through the OSF V2 API howeverthey could not be created or viewed via the OSF website.
+"""
+import sys
+import logging
+
+from website import settings
+from website.app import setup_django, init_app
+from scripts import utils as script_utils
+
+setup_django()
+from osf.models import OSFUser
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration(dry):
+    logger.warn('Removing invalid social entries!')
+    if dry:
+        logger.warn('Dry_run mode')
+    for user in OSFUser.find():
```

Comment:
I'm having trouble getting .only() to work. I haven't been able to find an example in the code base. I found the Django Doc on it but am receiving an error I haven't been able to resolve.
```
OSFUser.objects.only("social")
...
/usr/local/lib/python2.7/site-packages/django/db/models/query_utils.pyc in __get__(self, instance, cls)
    114             val = self._check_parent_chain(instance, self.field_name)
    115             if val is None:
--> 116                 instance.refresh_from_db(fields=[self.field_name])
    117                 val = getattr(instance, self.field_name)
    118             data[self.field_name] = val

TypeError: refresh_from_db() got an unexpected keyword argument 'fields'
```